pkg/converter/converter.go

```diff
@@ -120,6 +121,18 @@ func composeServiceToSecret(workload *ir.Service, refSlug string, labels map[str
 	return &secret
 }
 
+func composeServiceToPullSecret(authConf string, name string, labels map[string]string) *core.Secret {
+	secret := core.Secret{}
+	secret.APIVersion = "v1"
+	secret.Kind = "Secret"
+	secret.Type = core.SecretTypeDockerConfigJson
+	secret.Name = name + "-imagePull"
```

Comment:
We don't seem to be doing camel-case for k8s resource names generally, I'd prefer to avoid it here as well, and just all it "-imagepull" all lowercase.

Reply:
True, I went with no 2.